RNA-seq/setup/exercises/04-tximeta_medulloblastoma.R

```diff
@@ -0,0 +1,38 @@
+# J. Shapiro for ALSF CCDL 2021 (modified from J. Taroni)
+# Process SRP049821 RNA-seq data with tximeta for bulk RNA-seq exercise
+# notebook
+#
+# Command line usage: Rscript 04-tximeta_medulloblastoma.R
+# Input (paths hardcoded): quant files from SRP150101
+# Output (path hardcoded): output of tximeta (gene summarized)
+
+data_dir <- "/shared/data/training-modules/RNA-seq/data/medulloblastoma"
```

Comment:
`file.path()`?

Reply:
I kind of deliberately didn't use that here, because this script isn't meant to be too portable, with its hard coded paths. 🤷🏼